src/main/assets/_documentation.styl

```diff
@@ -1,4 +1,12 @@
 article
+
+  /* This takes into account the header when clicking anchor links */
+  :target:before
+    content ""
+    display block
+    height header-height
+    margin-top - header-height
+
```

Comment:
@rasummer Is this ok?  Basically, when you navigated to an anchor in the docs, the title would be chopped off by the sticky header, so I've added this to fix it, which seems to work to me, but you probably have a better idea of the best way to address it.